src/code-templates/libraries/logger/test/index.test.ts

```diff
@@ -1,31 +1,34 @@
 import sinon from 'sinon';
 import { logger } from '../index';
 
-beforeEach(() => {
-  sinon.restore();
-  logger.resetLogger();
-});
-
 describe('logger', () => {
-  test('When no explicit configuration is set, info logs are written', async () => {
-    // Arrange
-    const stdoutStub = sinon.stub(process.stdout, 'write');
+  let stdoutStub: sinon.SinonStubbedMember<typeof process['stdout']['write']>;
```

Comment:
@rluvaton 

I'm concerned with this line as it raises the reader's reading complexity, is there any value here?

Reply:
type safety, removed and recreate in every test